sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
 
+### Breaking Changes
+
+* Removed constructors in clients that takes a `Azure::Identity::ClientSecretCredential`.
```

Comment:
```suggestion
- Removed constructors in clients that takes a `Azure::Identity::ClientSecretCredential`.
```

Azure CORE and all other languages uses `-` instead of `*` for the markdown bullets.

Reply:
In Storage * is used universally. I think it is a bigger change that needs to be done in a separate PR.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1019